fixtures/exec-yaml-output.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: configs.flanksource.com/v1
+kind: ScrapeConfig
+metadata:
+  name: exec-yaml-output
+spec:
+  schedule: "@every 10m"
+  exec:
+    - name: cluster-info-yaml
+      type: Kubernetes::Cluster
+      id: $.metadata.name
+      name: $.metadata.name
```

YAMLlint (1.37.1)

[error] duplication of key "name" in mapping

(key-duplicates)